src/main/java/com/leetcode/algorithm/easy/SameTree

```diff
@@ -0,0 +1,13 @@
+class Solution {
+    public boolean isSameTree(TreeNode p, TreeNode q) {
+        
+        if(p==null && q==null)
+            return true;
+        if(p==null || q==null)
```

Comment:
You have the same condition logic at line 4. 

Thanks for trying it out this problem. 
dinitrogen-tetroxide already submitted his code and already merged so you may try different problem. 

And when you submit your code, make sure to add unit test as well. 
Thanks.